components/automate-deployment/pkg/server/automate_config.go

```diff
@@ -235,3 +254,139 @@ func (s *server) updateUserOverrideConfigFromRestoreBackupRequest(req *api.Resto
 
 	return nil
 }
+
+func setConfigForRedirectLogs(req *api.PatchAutomateConfigRequest, existingCopy *deployment.AutomateConfig) error {
+
+	//Set the config if already not set
+	if req.GetConfig().GetGlobal().GetV1().GetLog().GetRedirectSysLog().GetValue() == true &&
+		existingCopy.GetGlobal().GetV1().GetLog().GetRedirectSysLog().GetValue() == false {
+
+		err := createConfigFileForAutomateSysLog(req.GetConfig().GetGlobal().GetV1().GetLog().GetRedirectLogFilePath().GetValue())
+		if err != nil {
+			return status.Error(codes.Internal, "Failed to create configuration into syslog")
```

Comment:
here we are swallowing the actual error. If we would like to display an error message similar to this, then write the actual error message at least in the logs.